doc/api/deprecations.md

```diff
@@ -999,6 +999,15 @@ Type: Documentation-only
 The option `produceCachedData` has been deprecated. Use
 [`script.createCachedData()`][] instead.
 
+<a id="DEP00XX"></a>
+### DEP00XX: process.binding()
+
+Type: Documentation-only
+
+The `process.binding()` API is intended for use strictly by Node.js internal
+code to provide a bridge between Node.js' JavaScript and native code layer.
```

Comment:
Since `Node.js` is singular, it should be `Node.js's`. But really, might be best to avoid the whole possessive entirely by just removing a couple words so it becomes: `bridge between JavaScript and native code`.

Reply:
Even better perhaps is remove everything after `code` from this sentence. You don't have to explain what the function does. That's superfluous information in this case.